### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by ImNotStable**

### CODE REVIEW

**Comment by ImNotStable** [comment]
Look over all changes and ask about anything you don't understand please.

File: .idea/jarRepositories.xml [hunk 1]
```diff
@@ -6,6 +6,16 @@
...
     </remote-repository>
+    <remote-repository>
+      <option name="id" value="jitpack.io" />
+      <option name="name" value="jitpack.io" />
+      <option name="url" value="https://jitpack.io" />
+    </remote-repository>
+    <remote-repository>
+      <option name="id" value="github" />
+      <option name="name" value="github" />
+      <option name="url" value="https://github.com/ImNotStable/QualityEconomy" />
```
Comment (Tauses) [inline .idea/jarRepositories.xml:17]: Why did you hook it to your economy plugin?

File: .idea/jarRepositories.xml [hunk 2]
```diff
@@ -6,6 +6,16 @@
       <option name="name" value="Central Repository" />
       <option name="url" value="https://repo.maven.apache.org/maven2" />
     </remote-repository>
+    <remote-repository>
+      <option name="id" value="jitpack.io" />
+      <option name="name" value="jitpack.io" />
+      <option name="url" value="https://jitpack.io" />
```
Comment (Tauses) [inline .idea/jarRepositories.xml:12]: What is this needed for?

File: .idea/jarRepositories.xml [hunk 3]
```diff
@@ -26,5 +36,15 @@
       <option name="name" value="JBoss Community repository" />
       <option name="url" value="https://repository.jboss.org/nexus/content/repositories/public/" />
     </remote-repository>
+    <remote-repository>
+      <option name="id" value="central" />
+      <option name="name" value="central" />
+      <option name="url" value="https://repo1.maven.org/maven2" />
+    </remote-repository>
+    <remote-repository>
```
Comment (Tauses) [inline .idea/jarRepositories.xml:44]: I suppose this is for the dependency?

File: .idea/uiDesigner.xml
Comment (Tauses) [inline .idea/uiDesigner.xml:1]: Is this for the GUI?

File: pom.xml [hunk 4]
```diff
@@ -12,7 +12,7 @@
     <name>HuskHomes GUI</name>
 
     <properties>
-        <java.version>1.8</java.version>
+        <java.version>17</java.version>
```
Comment (Tauses) [inline pom.xml:15]: How can I automate my IDE so it does this?

File: pom.xml [hunk 5]
```diff
@@ -42,6 +42,22 @@
                         </configuration>
                     </execution>
                 </executions>
+                <configuration>
+                    <relocations>
+                        <relocation>
+                            <pattern>fr.mrmicky.fastinv</pattern>
```
Comment (Tauses) [inline pom.xml:48]: No clue what this is.

File: pom.xml [hunk 6]
```diff
@@ -61,6 +77,10 @@
             <id>sonatype</id>
             <url>https://oss.sonatype.org/content/groups/public/</url>
         </repository>
+        <repository>
+            <id>william278.net</id>
+            <url>https://repo.william278.net/releases</url>
```
Comment (Tauses) [inline pom.xml:82]: Is this for the dependency ?

**Review by Tauses — APPROVED** [review]
Look through my comments on your changes please. :) 👍

File: .idea/jarRepositories.xml (hunk 1 above)
Comment (ImNotStable) [reply .idea/jarRepositories.xml:17]: This is a mistake I'll figure out how to resolve this, I'm unsure of the exact cause.

File: .idea/jarRepositories.xml (hunk 2 above)
Comment (ImNotStable) [reply .idea/jarRepositories.xml:12]: Very commonly needed repository but I'll remove this as well.

File: .idea/jarRepositories.xml (hunk 3 above)
Comment (ImNotStable) [reply .idea/jarRepositories.xml:44]: Just maven central which many dependencies use

File: .idea/uiDesigner.xml
Comment (ImNotStable) [reply .idea/uiDesigner.xml:1]: All changes within .idea are unrelated to the code.

File: pom.xml (hunk 4 above)
Comment (ImNotStable) [reply pom.xml:15]: I did this myself, I use 17

File: pom.xml (hunk 5 above)
Comment (ImNotStable) [reply pom.xml:48]: This is an inventory API very light weight and better than using Bukkit directly.

File: pom.xml (hunk 6 above)
Comment (ImNotStable) [reply pom.xml:82]: Yes this is for HuskHomes itself

**Pull request merged**